quic/s2n-quic-qns/etc/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN set -eux; \
   sed -i '/xdp/d' quic/s2n-quic-platform/Cargo.toml; \
   sed -i '/xdp/d' quic/s2n-quic-qns/Cargo.toml; \
   sed -i '/xdp/d' quic/s2n-quic/Cargo.toml; \
-  rm -rf quic/s2n-quic-bench quic/s2n-quic-events quic/s2n-quic-sim quic/s2n-quic-tests
+  rm -rf quic/s2n-quic-bench quic/s2n-quic-sim quic/s2n-quic-tests
```

Comment:
The `quic` directory is copied above, and s2n-quic-events was removed from it. Now that s2n-events isn't in `quic`, it no longer needs to be removed.